cliconfig/config.go

```diff
@@ -15,19 +15,19 @@ import (
 )
 
 const (
-	// Where we store the config file
-	CONFIGFILE     = "config.json"
-	OLD_CONFIGFILE = ".dockercfg"
+	// CONFIGFILE is the name of config file
+	CONFIGFILE    = "config.json"
```

Comment:
weird golint don't highlight this one :/ probably because it's not underscored, I'd prefer to have it called `ConfigFile` then, wdyt?